docs/cromwell_features/RetryWithMoreMemory.md

```diff
@@ -0,0 +1,35 @@
+#Retry with More Memory
+
+With this feature one can specify an array of strings which when encountered in the `stderr` file by Cromwell, 
+allows the task to be retried with more memory. The retry will be counted against the `maxRetries` count mentioned in 
+the `runtimeAtrributes` in the task. There are 2 settings for this feature:
+* `system.memory-retry-error-keys` : the error keys that need to be set in Cromwell config
```

Comment:
RTD is more picky about MD grammar than github. You'll want to add a newline before the bullet point list